test/hotspot/jtreg/vmTestbase/nsk/jvmti/SetFieldModificationWatch/setfmodw001/TestDescription.java

```diff
@@ -47,11 +47,3 @@
  *          /test/lib
  * @run main/othervm/native -agentlib:setfmodw001 nsk.jvmti.SetFieldModificationWatch.setfmodw001
  */
-
-/*
- * @test id=logging
- *
- * @library /vmTestbase
- *          /test/lib
- * @run main/othervm/native -agentlib:setfmodw001 -XX:TraceJVMTI=ec+,+ioe,+s -Xlog:jvmti=trace:file=vm.%p.log nsk.jvmti.SetFieldModificationWatch.setfmodw001
- */
```

Comment:
What's the reason for removing this test run?

Reply:
I was not able to reproduce failure locally at the beginning. So I added this test run just to get more information about failure from CI runs for initial analysis. 
The information from logging clearly pointed that we just never got event while set Thread-1 to interprter_only mode correctly. 
However, it is not needed now so I am removing it.